manager/src/main/java/com/bakdata/quick/manager/graphql/GraphQLConverter.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *    Copyright 2022 bakdata GmbH
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.bakdata.quick.manager.graphql;
+
+import com.bakdata.quick.common.graphql.GraphQLUtils;
+import com.bakdata.quick.common.type.QuickTopicType;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.RuntimeWiring;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+import io.confluent.kafka.schemaregistry.ParsedSchema;
+
+/**
+ * An interface for converting GraphQL schema to a {@link ParsedSchema} object.
```

Comment:
```suggestion
 * Converter for transforming a GraphQL schema to a {@link ParsedSchema} object.
```
Just as an example, similar to the comment above